keep/actions/actions_exception.py

```diff
@@ -0,0 +1,5 @@
+class ActionsFactoryException(Exception):
```

Comment:
you could use httpexception from fastapi which basically get message (details) and status_code

Reply:
I just extended the Actions Exception with `HTTPException` from `fastapi`. This will also help to detect exception coming from `Actions`

```
from fastapi import HTTPException

class ActionsCRUDException(HTTPException):
    """An exception class that depicts any error comming from Action"""
```